Examples/Examples/FeatureFormExampleView.swift

```diff
@@ -170,24 +141,63 @@ extension FeatureFormExampleView {
     /// A Boolean value indicating whether general form workflow errors are presented.
     private var alertIsPresented: Binding<Bool> {
         Binding {
-            self.alertError != nil
+            submissionError != nil
         } set: { newAlertIsPresented in
             if !newAlertIsPresented {
-                self.alertError = nil
+                submissionError = nil
             }
         }
     }
     
+    /// The feature form view shown in the sheet over the map.
+    private var featureFormView: some View {
+        FeatureFormView(root: featureForm!, isPresented: featureFormViewIsPresented)
+            .onFormEditingEvent { editingEvent in
+                if case .savedEdits = editingEvent,
+                   let table = featureForm?.feature.table as? ServiceFeatureTable,
+                   !editedTables.contains(where: { $0 === table }) {
+                    editedTables.append(table)
+                }
+            }
+    }
+    
     /// A Boolean value indicating whether the form is presented.
     private var featureFormViewIsPresented: Binding<Bool> {
-        .init {
+        Binding {
             featureForm != nil
         } set: { newValue in
             if !newValue {
                 featureForm = nil
             }
         }
     }
+    
+    /// The button used to apply edits made in forms.
+    @ViewBuilder private var submitButton: some View {
+        let databases = editedTables.compactMap(\.serviceGeodatabase)
+        let localEditsExist = databases.contains(where: \.hasLocalEdits)
+        if !featureFormViewIsPresented.wrappedValue, localEditsExist {
+            Button("Submit") {
+                Task {
+                    await applyEdits()
+                }
+            }
+        }
+    }
+    
+    /// Overlay content that indicates the form is being submitted to the user.
+    @ViewBuilder private var submittingOverlay: some View {
+        if editsAreBeingApplied {
+            HStack(spacing: 5) {
+                ProgressView()
+                    .progressViewStyle(.circular)
+                Text("Submitting")
```

Comment:
`ProgressView` has an initializer for this:

```suggestion
                ProgressView("Submitting")
```